transformers/events/spot_file/par/initializer/initializer.go

```diff
@@ -0,0 +1,14 @@
+package initializer
+
+import (
+	"github.com/makerdao/vdb-mcd-transformers/transformers/events/spot_file/par"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/libraries/shared/transformer"
+)
+
+var EventTransformerInitializer transformer.EventTransformerInitializer = event.Transformer{
+	Config:    shared.GetEventTransformerConfig(constants.SpotFileParTable, constants.SpotFileParSignature()),
+	Converter: &par.Converter{},
```

Comment:
Are we moving away from using pointers here? Tbh I don't know why we'd use one or the other in this instance, just thinking about consistency

Reply:
Yeah good catch - I think we should trend toward only using them when we have a good reason, like needing to modify the internal state of the struct. Will remove here since we pass the db connection as an arg to the function (rather than calling `SetDB` on the converter)